ush/retrieve_data.py

```diff
@@ -897,10 +954,17 @@ def parse_args(argv):
     # Required
     parser.add_argument(
         "--anl_or_fcst",
```

Comment:
The name of this flag should be changed to reflect the new options. Maybe `file_set` or something descriptive? The help entry should also be updated to describe all the different options.